features/hooks/filtering.feature

```diff
@@ -244,3 +244,186 @@ Feature: filters
       .after context
       """
 
+  Scenario: Filtering hooks using symbols
```

Comment:
This is to flesh out the cuke.  Not having specific examples here made these filters a (previously) hidden gem of rspec, which are very cool and powerful.  I added the documentation I wish I had when we first started using these filters.